src/databox/HISTORY.rst

```diff
@@ -3,6 +3,13 @@
 Release History
 ===============
 
+0.1.4
+++++++
+* `az databox job create`: Add new parameter `--transfer-type` `--transfer-configuration-type` `--transfer-filter-details` `--data-box-customer-disk` to support managing import or export` jobs
+* `az databox job create/update`: Add new parameter `--kek-type` `--kek-identity` `--kek-url` `--kek-vault-resource-id` to support managing customer managed key
+* `az databox job`: Add new subcommand `mark-devices-shipped` to support marking devices shipped
+* Migrate `az databox` commands to aaz
```

Comment:
Remove "Migrate `az databox` commands to aaz" as it's just an internal implementation detail.